jib-core/src/main/java/com/google/cloud/tools/jib/api/Containerizer.java

```diff
@@ -145,16 +146,20 @@ public static Containerizer to(TarImage tarImage) {
   @Nullable private Path applicationLayersCacheDirectory;
   @Nullable private EventHandlers eventHandlers;
   private boolean allowInsecureRegistries = false;
+  private boolean offline = false;
+  private boolean mustBeOnline;
```

Comment:
nit: let's be consistent by saying `= false`.

Reply:
Gives a warning for redundant initialization, I guess since it's a parameter in the constructor.

Reply:
I guess I should make it final then.